src/main/java/com/cloudcomputing/cloudkarma/RemainingResources.java

```diff
@@ -0,0 +1,46 @@
+package com.cloudcomputing.cloudkarma;
+
+public class RemainingResources {
+
+	public class FreeSpace {
+
+		int cpu_avaliable;
+		int memory_avaliable;
```

Comment:
please follow camelCasing for variable names and 'available' spelling

Reply:
Done